run-code-climate-on-tags/main.py

```diff
@@ -116,7 +116,7 @@ def _write_metadata(file_name: Path, src_root: str, tag: str):
 
 
 def _write_results(file_name: Path, results: Dict):
-    file_location = Path("output") / Path("frequencies") / file_name
+    file_location = Path("output") / Path("outcome") / file_name
```

Comment:
```suggestion
    file_location = Path("output") / Path("violations") / file_name
```
Pretty sure our modu uses "violations" not outcome